hotspot/src/os/aix/vm/os_aix.cpp

```diff
@@ -1452,7 +1452,7 @@ bool os::dll_address_to_library_name(address addr, char* buf,
 
 // Loads .dll/.so and in case of error it checks if .dll/.so was built
 // for the same architecture as Hotspot is running on.
-void *os::dll_load(const char *filename, char *ebuf, int ebuflen) {
+static void* dll_load(const char *filename, char *ebuf, int ebuflen) {
```

Comment:
Are you sure this is correct? The [JDK 11u](https://github.com/openjdk/jdk11u-dev/commit/6808c5c89d1e30f481ebe444f93175232bcd30e2#diff-5f68fc3ce77d56a6e5a398c51eb1e1b56ea41175e8a86a1c5d92e167ba2b2b40R1316) change renamed this to `dll_load_library`. Does this even work?

Reply:
@jerboaa  Still working on it. 11 and 17 backports were very much clean and had no issues. For some reason jdk8 compilation is failing in my local. So hopefully i will get it working soon.

Reply:
@jerboaa  thanks for the catch. That might be causing it. I had to make changes in a file located in different location , so the backport didn't apply clean.

Reply:
@jerboaa I see the certificate tests failing in the tests. Are these due to some other issue ? I do not see how my fix would cause a certificate issue.

Reply:
@suchismith1993 Please merge latest master, those tests should be on the problem list by now.

Reply:
@jerboaa So i need to rebase then right ?